### DISCUSSION TITLE
13 giugno 2023

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/mdp2223#7] [Q&A] [answered]
In questa discussion a tema la traccia di esame del 13 giugno 2023 discuteremo di eventuali proposte di soluzione all'esame in questione

**Comment by Gio0203c** [comment, marked as answer]
[13-06-2023 - COMPLETO.pdf](https://github.com/sapienzastudentsnetwork/mdp2223/files/12988542/13-06-2023.-.COMPLETO.pdf)

**Comment by Frisbon** [comment]
Traccia senza soluzione
https://drive.google.com/file/d/1RXh97oTBXkgkPmrbeV7mgIsLL6cwUsOf/view?usp=sharing

**Comment by FeddyLix17** [comment]
[Walkthrough Esame 13 Giugno 2023](https://github.com/FeddyLix17/Computer-Science/tree/main/Primo%20Anno/Secondo%20Semestre/Metodologie%20di%20Programmazione/Esami%20precedenti/2023/Giugno/Esame%2013%20Giugno%202023)
